packages/node/src/models/free-balance.ts

```diff
@@ -273,6 +282,9 @@ function serializeFreeBalanceState(
 export function convertCoinTransfersToCoinTransfersMap(
   coinTransfers: CoinTransfer[]
 ): CoinTransferMap {
+  if (!coinTransfers) {
```

Comment:
@cf19drofxots Instead of throwing an error, wouldn't it be best to default `coinTransfers` to `[]`?

Reply:
The expectation is that this function should always be called with a list of length non-zero, so even if we default to `[]` that's a silent ignoring of whoever called this having the wrong argument value, hence the throw.